### ISSUE TITLE
Rename "doc" folder to "docs"

### ISSUE THREAD

**Issue by koppor** [adoble/adr-j#18]
I would propose to rename `doc` to `docs` (at least in this repository) to enable rendering of the docs using gh-pages (see https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages) and to be consistent to other GitHub repositories.

I know that adr-tools rejected that proposal: https://github.com/npryce/adr-tools/issues/39 - maybe, you can spend a thought regarding that, too.

**Comment by adoble** [comment]
- The reason for the [rejection](https://github.com/npryce/adr-tools/issues/39) was that it was not backward compatible. This doesn’t apply to adr-j  as the directories used for the ADRs is stored in the `adr.properties` file (`docPath`). This means that users do not have to do anything with previously initialized projects.

- #13 could mean that if users really wanted to change from `doc `to `docs ` in an existing project they could do so. Note, the `config `command is more complicated than I first thought as config changes imply making a lot of changes to existing projects. Not impossible, it's just that I have not got around to it. 

- I would prefer that we use in this repository the default. Changing to `docs `means a bit of fiddling around (a test for the config command?).

-  As we are now using semantic versioning , have to consider if this change is an "incompatible API change" . In which case I would prefer to wait until we have accrued enough changes (e.g. #4, #13 plus anything else that's wanted) to justify a 4.0.0 release. What do you think? 

In short, I'm sympathetic to the idea, just need consider the above.

**Comment by adoble** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue 18 - Rename "doc" folder to "docs"](https://trello.com/c/uUquebAg/40-issue-18-rename-doc-folder-to-docs)

**Comment by adoble** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue 18 - Rename "doc" folder to "docs"](https://trello.com/c/uUquebAg/40-issue-18-rename-doc-folder-to-docs)

**Comment by adoble** [comment]
This issue has been open for a long time now. Personally, I still think it is a good idea, but as it is a change to the user facing API I'm still deferring it until we get to version 4 which is a major effort requiring changes to how `adr-j `fundamentally works. 

As such I'm still leaving it open..
